### ISSUE TITLE
Build failed with new version of mkdocs in Circle CI

### ISSUE THREAD

**Issue by arturalbov** [squidfunk/mkdocs-material#830]
## Description

Build failed with new version of mkdocs in Circle CI through docker image.
On version 2.9.2 everything was OK.
After version 2.9.4 build broken.

Logs:
```
#!/bin/bash -eo pipefail
cd knowledge
mkdocs gh-deploy -m "deployed $CIRCLE_SHA1 with mkdocs [skip ci]"

INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /root/build/knowledge/site 
WARNING -  Documentation file 'cyber-markets/contributing/contributing.md' contains a link to 'CHANGELOG.md' which does not exist in the documentation directory. 
WARNING -  Documentation file 'cyber-search/contributing/contributing.md' contains a link to 'CHANGELOG.md' which does not exist in the documentation directory. 
WARNING -  Documentation file 'сhaingear/development.md' contains a link to 'сhaingear/remix.ethereum.org' which does not exist in the documentation directory. 
ERROR   -  Error building page 'index.md': object of type 'NoneType' has no len() 
Traceback (most recent call last):
  File "/usr/bin/mkdocs", line 11, in <module>
    load_entry_point('mkdocs==1.0rc1', 'console_scripts', 'mkdocs')()
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/mkdocs-1.0rc1-py2.7.egg/mkdocs/__main__.py", line 186, in gh_deploy_command
    build.build(cfg, dirty=not clean)
  File "/usr/lib/python2.7/site-packages/mkdocs-1.0rc1-py2.7.egg/mkdocs/commands/build.py", line 292, in build
    _build_page(file.page, config, files, nav, env, dirty)
  File "/usr/lib/python2.7/site-packages/mkdocs-1.0rc1-py2.7.egg/mkdocs/commands/build.py", line 213, in _build_page
    output = template.render(context)
  File "/usr/lib/python2.7/site-packages/Jinja2-2.10-py2.7.egg/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python2.7/site-packages/Jinja2-2.10-py2.7.egg/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/build/knowledge/theme_extend/material/main.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/root/build/knowledge/theme_extend/material/base.html", line 112, in top-level template code
    {% include "partials/tabs.html" %}
  File "/root/build/knowledge/theme_extend/material/partials/tabs.html", line 9, in top-level template code
    {% include "partials/tabs-item.html" %}
  File "/root/build/knowledge/theme_extend/material/partials/tabs-item.html", line 15, in top-level template code
    {% if (nav_item.children | first).children | length > 0 %}
TypeError: object of type 'NoneType' has no len()
Exited with code 1
```

Job config:
```
version: 2

workflows:
  version: 2
  default_workflow:
    jobs:
      - update_doc:
          filters:
            branches:
              only: master

jobs:
  update_doc:
    working_directory: ~/build
    docker:
      - image: squidfunk/mkdocs-material:2.9.2
    steps:
      - checkout
      - add_ssh_keys:
          fingerprints:
            - 5f:5b:bf:dc:2b:99:a1:1d:44:61:b3:26:e5:f6:cc:11
      - run:
          name: Update documentation
          command: |
            cd knowledge
            mkdocs gh-deploy -m "deployed $CIRCLE_SHA1 with mkdocs [skip ci]"
```

### Expected behavior

Build passes

### Actual behavior

Build failing

### Steps to reproduce the bug

Run job in Circle CI with config above.

### Package versions

* Python: 2.7
* MkDocs: 2.9.4
* Material: IDK

### Project configuration

---

### System information

* OS: ---
* Browser: ---

**Comment by squidfunk** [comment]
Please check the tabs item partial - I got the same error and had to make changes in 2.9.4 which needs to be included in extended partials

**Comment by squidfunk** [comment]
Could you actually just try to pull `2.9.4` again and try to rebuild? It may be fixed through 7ef1dd08f8aff842d4ac2c5f290de649dcf2d3df

**Comment by arturalbov** [comment]
Yes, it works. Thanks @squidfunk !
